bindings/cffirmware.i

```diff
@@ -133,12 +133,13 @@ void collisionAvoidanceUpdateSetpointWrap(
     free(workspace);
 }
 
-void state_set_neighbor_position(state_t *state, int idx, uint8_t id, float x, float y, float z)
+void state_set_neighbor_position(state_t *state, int idx, uint8_t id, float x, float y, float z, uint8_t num_neighbors)
 {
     state->neighbors[idx].id = id;
     state->neighbors[idx].pos.x = x;
     state->neighbors[idx].pos.y = y;
     state->neighbors[idx].pos.z = z;
+    state->num_neighbors = num_neighbors;
```

Comment:
This shouldn't be needed, since num_neighbors is not in an array. You can directly assign in the Python code.

Reply:
Done.